### PR TITLE
Created login/signup page

### PR DESCRIPTION
**Pull request by j24m**
Hello I have created login/signup page. Kindly review.

### CODE REVIEW

File: login.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.disclaimer{
+    font-size: 1.5rem;
+    text-align: center;
```
Comment (dipen2000) [inline login.css:3]: Make a class for stuff like this so you can re use it wherever you want.

**Review by dipen2000 — APPROVED** [review]
Yep looks good , i don't have much to say in this one. Standard HTML stuff. Good to go.

File: login.html [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                            <input type="email" class="form-input-box" placeholder="Enter your Email id" required>
+                            <input type="password" class="form-input-box" placeholder="Password">
+                            <button type="submit" class="submit-btn">
+                                Submit
+                            </button>
+                            <div class="checkbox-remember-me">
+                                <input type="checkbox"><span>Remember me</span>
+                            </div>
+                        </form>
+                        <button type="button" class="btn-new-here" onclick="openRegister()">I'm New Here</button>
```
Comment (rahulyadav139) [inline login.html:94]: The login/signup page is looking superb. Animation is very subtle and amazing looking. Just ignore to put javascript code in the HTML file; place it inside a separate javascript file.  There is no hard rule but we should separate logic and skeleton codes in a separate file. Other than that everything is great. Awesome work.

**Review by rahulyadav139 — APPROVED** [review]

**Review by rahulk31 — APPROVED** [review]
UI Suggestion: You can plan to give some padding below in next iteration of your Ecommerce.
image


Overall, I liked the way you have made use of the Component Library and less explicit css. Good work. Can be merged. 👍

File: login.css (hunk 1 above)
Comment (j24m) [reply login.css:3]: That's a personal preference so maybe I will do it later.

**Pull request merged**